src/raft.cpp

```diff
@@ -92,11 +92,7 @@ coord_t Raft::getZdiffBetweenRaftAndLayer1()
         return 0;
     }
     const coord_t airgap = std::max(coord_t(0), train.settings.get<coord_t>("raft_airgap"));
-    const coord_t layer_0_overlap = mesh_group_settings.get<coord_t>("layer_0_z_overlap");
-
-    const coord_t layer_height_0 = mesh_group_settings.get<coord_t>("layer_height_0");
-
-    const coord_t z_diff_raft_to_bottom_of_layer_1 = std::max(coord_t(0), airgap + layer_height_0 - layer_0_overlap);
+    const coord_t z_diff_raft_to_bottom_of_layer_1 = std::max(coord_t(0), airgap);
```

Comment:
> I was wondering if this comment still makes sense with the proposed changes

Both the comment and the function name (layer 1 as opposed to layer0) make no sense any more.

The code still works because layers with n>0 are shifted downwards elsewhere (search for `// add the raft offset to each layer` in `FffPolygonGenerator.cpp`).

This doesn't touch the air-gap in any way, and this function is only referenced within `raft.cpp`. I think the comment should be altered/removed, and the function name changed.


> airgap is already the max between 0 and the setting value. There is no need here to perform another max

Yeah fair. I'll ad it to the batch.